### PR TITLE
[python] add mediatype to musicinfotag

### PR DESCRIPTION
**Pull request by ronie**
allow to retrieve the mediatype from the musicinfotag of a listitem.

### CODE REVIEW

**Comment by ronie** [comment]
jenkins build this please

**Comment by ronie** [comment]
jenkins build this please

File: xbmc/interfaces/legacy/InfoTagMusic.h [hunk 1]
```diff
@@ -103,6 +103,30 @@ namespace XBMCAddon
...
+      /// @brief \python_func{ getMediaType() }
+      ///-----------------------------------------------------------------------
+      /// Get the media type of the music item.
+      ///
+      /// @return [string] media type
+      ///
+      /// Available strings about media type for music:
+      /// | String         | Description                                       |
+      /// |---------------:|:--------------------------------------------------|
+      /// | artist         | To define it as an artist
```
Comment (phil65) [inline xbmc/interfaces/legacy/InfoTagMusic.h]: these docs are needed for setting mediatype, but not for getting mediatype i guess?

Comment (ronie) [inline xbmc/interfaces/legacy/InfoTagMusic.h]: we have these docs for video mediatype as well:
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/legacy/InfoTagVideo.h#L253-L268

if addons are to interpreted to return value, i don't think it's bad to list the possible values.

Comment (phil65) [inline xbmc/interfaces/legacy/InfoTagMusic.h]: Ok. "To define it as an artist" more sounds like sth for setting mediatype though.
Should we remove mediatype "music" from core btw? Atm i have no idea what that should be used for..

Comment (ronie) [inline xbmc/interfaces/legacy/InfoTagMusic.h]: description updated.

sure, i've no problem with dropping the music mediatype.

**Comment by ronie** [comment]
jenkins build this please

**Review by phil65 — APPROVED** [review]

**Comment by ronie** [comment]
jenkins build this please

**Comment by ronie** [comment]
jenkins build this please

**Pull request merged**